setup.py

```diff
@@ -309,10 +309,7 @@ def build_libs(libs):
     if not IS_WINDOWS:
         if USE_NINJA:
             my_env["CMAKE_GENERATOR"] = '-GNinja'
-            my_env["CMAKE_INSTALL"] = 'ninja install'
-        else:
-            my_env['CMAKE_GENERATOR'] = ''
-            my_env['CMAKE_INSTALL'] = 'make install'
+            my_env["CMAKE_INSTALLER"] = 'ninja'
```

Comment:
uhh... some of us don't have ninja installed?

Reply:
it's still under the `if`, and if this is unset then the layer below (build_pytorch_libs.sh) defaults to `make`